Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -382,10 +382,10 @@ PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f,
 
         amrex::Real weight;
         if (q_tot != 0.0){
-            weight = q_tot/(p_q*amrex::Real(npart));
+            weight = abs(q_tot)/(abs(charge)*amrex::ParticleReal(npart));
```

Comment:
This somehow picks up an AMReX int function.
Instead include
```C++
#include <cmath>
```
and
```suggestion
            weight = std::abs(q_tot)/(std::abs(charge)*amrex::ParticleReal(npart));
```